.github/workflows/build.yml

```diff
@@ -31,6 +31,10 @@ jobs:
         if: always()
         run: ./scripts/fmt.sh
 
+      - name: Generate README.md
```

Comment:
I think the fmt script already does this, because `make fmt` also re-generates the README.md based on the gotmpl https://github.com/cdr/enterprise-helm/blob/d51e11a277291eb192b707affd2befef51b3a289/scripts/fmt.sh#L18

It maybe isn't clear that's happening though, open to renaming `make fmt` to something else!

Reply:
Ahh... nah it was just my bad for not reading tbh. I'll remove the new script!

Reply:
You can confirm this is happening by looking at the build output, btw:

![image](https://user-images.githubusercontent.com/52710/119242608-2afe9700-bb14-11eb-8c06-2ed93560adc0.png)

Reply:
We probably should have had separate `make fmt`/`make gen` steps, and set up `make fmt` to depend on make gen... I was just being lazy lol